deploy/operator.yaml

```diff
@@ -15,8 +15,7 @@ spec:
       serviceAccountName: project-initialize
       containers:
         - name: project-initialize
-          # Replace this with the built image name
-          image: REPLACE_IMAGE
+          image: "quay.io/cnuland/project-init-operator:latest"
```

Comment:
@cnuland did you intend to leave your personal repository as the source for the operator?

Reply:
Yes, for now until we create a CI/CD pipeline for this and a COP org repo on quay. This operator is still in an early alpha, so I'm personally okay with it.